x-pack/plugins/apm/server/lib/service_map/get_service_map_service_node_info.ts

```diff
@@ -71,32 +73,24 @@ export async function getServiceMapServiceNodeInfo({
   };
 }
 
-async function getErrorMetrics({ setup, minutes, filter }: TaskParameters) {
-  const { client, indices } = setup;
-
-  const response = await client.search({
-    index: indices['apm_oss.errorIndices'],
-    body: {
-      size: 0,
-      query: {
-        bool: {
-          filter: filter.concat({
-            term: {
-              [PROCESSOR_EVENT]: 'error',
-            },
-          }),
-        },
-      },
-      track_total_hits: true,
-    },
+async function getErrorMetrics({
+  environment,
+  serviceName,
+  setup,
+}: {
+  environment?: string;
+  serviceName: string;
+  setup: Options['setup'];
+}) {
+  const groupId = undefined;
+  const { average, noHits } = await getErrorRate({
+    serviceName,
+    groupId,
```

Comment:
Isn't this optional?
```suggestion
```

Reply:
I think the signature of the function changed on the branch I was working on. Good catch.

Reply:
Fixed.